packages/node_modules/@webex/plugin-meetings/src/meetings/index.js

```diff
@@ -640,6 +644,18 @@ export default class Meetings extends WebexPlugin {
                   createdMeeting.on(EVENTS.DESTROY_MEETING, (payload) => {
                     this.destroy(createdMeeting, payload.reason);
                   });
+
+                  createdMeeting.on(EVENTS.REQUEST_UPLOAD_LOGS, (meetingInstance) => {
+                    if (this.config.autoUploadLogs) {
+                      this.uploadLogs({
+                        callStart: meetingInstance.locusInfo && meetingInstance.locusInfo.fullState ? meetingInstance.locusInfo.fullState.lastActive : undefined,
```

Comment:
Let's clean this up with optional chaining:

```suggestion
                        callStart: meetingInstance.locusInfo?.fullState?.lastActive,
```

Reply:
agreed